### ISSUE TITLE
Icon request: Next.JS alternative color

### ISSUE THREAD

**Issue by AmodeusR** [devicons/devicon#911]
### What would you like to share?

Hello, I was going around using Next.JS icons to display on my readme profile, and I noticed the only two colors availables are white and black and no other alternative.

Having a colored alternative would be great in occasions we want to use such icons in a website that has light and dark themes. With only black and white colors available, the visibility of Next.JS's icons end up needing to be sacrificed in one of the themes.

**Comment by amacado** [comment]
Hello! You can basically give each icon any color you want it to have.
```
<i class="devicon-nuxtjs-plain-wordmark my-color"></i>

i.my-color {
  color: #FF00FF;
  font-size: 300px;
}
```
See this example: https://codepen.io/amacado/pen/wvqYMBK

**Comment by AmodeusR** [comment]
> Hello! You can basically give each icon any color you want it to have.
> 
> ```
> <i class="devicon-nuxtjs-plain-wordmark my-color"></i>
> 
> i.my-color {
>   color: #FF00FF;
> }
> ```
> 
> See this example: https://codepen.io/amacado/pen/wvqYMBK

I know it's possible to colorize it through some methods, but as I specified, I was applying it to Github's readme profile, which doesn't have css, nor supports svg tags. To make things more easy and practical, there could be a colored alternative to use in such cases, without the need to download, edit and reupload somewhere else to make it work in such environments.

**Comment by amacado** [comment]
I don't think we will accept an alternative color for the Next.js icon since the original icon is black (https://nextjs.org/), it's the original color of the logo. @AmodeusR which alternative color are you proposing? And for what reason?

**Comment by AmodeusR** [comment]
Well, about the reason, I already answered that. If you try to use it, for example, on your Github Profile, it'll be hard to see Nextjs's icon on the dark theme using the black logo, and if you try to counter that using the white icon, it'll be hard to see on the light theme.

There is no easy way to solve it other than manually downloading, changing color and reuploading the logo somewhere else.

About the color, although that's quite a subjective of a topic, I think rebecca purple really blends well with Next.js's logo. Thanks to @maltejur we can have a preview of [how it would look like](https://d2ybq8ooy8ehk5.cloudfront.net/nextjs/plain?color=rebeccapurple&size=500px).

 But since Maltejur also gave an excellent solution to such matter, if it works, I don't think it'll be something to worry about.

Sorry for the late reply and any inconvenience. Feel free to close this issue, since we finished discussing it, right?
